presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestDriver.java

```diff
@@ -73,23 +87,37 @@ public void setup()
         server.createCatalog(TEST_CATALOG, "tpch");
         server.installPlugin(new BlackHolePlugin());
         server.createCatalog("blackhole", "blackhole");
-
+        waitForNodeRefresh(server);
         setupTestTables();
     }
 
+    private static void waitForNodeRefresh(TestingPrestoServer server)
```

Comment:
I'd like to figure out why this occurs instead of adding this hack. It could be a bug somewhere that should be fixed in a better way.